implementations/rust/vault/src/types.rs

```diff
@@ -380,6 +380,46 @@ impl PublicKey {
         use PublicKey::*;
         matches!(self, P256(..))
     }
+
+    /// Return the byte type for the enum value
```

Comment:
```suggestion
    /// Return the byte type for the enum value
    // TODO: consider using discriminant values when supported in Rust: https://github.com/rust-lang/rust/issues/60553
```

Reply:
it would be preferable if/when supported to be able to use the enum as a `u8` instead of doing this match, which currently you must do. just to remind us in the future...